### DISCUSSION TITLE
Limitations for Concurrent Workers per repo or per organization?

### DISCUSSION THREAD

**Discussion by kevo1ution** [community/community#26343] [Actions]
<p>According to the <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits" rel="noopener nofollow ugc">usage limits docs</a>, the “Free” plan allows 20 total concurrent jobs.</p>
<p>Does this limitation apply per open source repository or per user/organization? So if I create two repositories in the same organization do they both get 20 jobs each or 20 jobs total?</p>

**Comment by kevo1ution** [comment, marked as answer]
<p>This was answered here: <a href="https://github.com/github/docs/discussions/1724#discussioncomment-146788" rel="noopener nofollow ugc">https://github.com/github/docs/discussions/1724#discussioncomment-146788</a>. The number of total concurrent jobs provided is on a per billing entity (e.g. user, organization) basis.</p>
<p>So in the example of two repositories in the same organization. If it has the “Free” Plan then the repositories share the 20 total workers (they do NOT get 20 repositories each).</p>

**Reply by gladykov** [reply]
Link leads to nowhere :/
